dist/index.html

```diff
@@ -133,7 +52,115 @@ <h3 class="text-c100 text-md py-3 font-bold">We Nourish</h3>
+        <div class="flex justify-between mx-auto my-20 flex-wrap">
+          <div class="sponser">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+          <div class="sponser">
+            <img src="assets/img/sponsers/envato-dark.png" alt="" />
+          </div>
+          <div class="sponser">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+          <div class="sponser">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+          <div class="sponser">
+            <img src="assets/img/sponsers/envato-light.png" alt="" />
+          </div>
+        </div>
+      </div>
+      <!-- Contact information -->
+      <div class="contact-section bg-c100 py-10">
+        <div class="container">
+          <div class="flex justify-between align-middle py-4 text-c000">
+            <!-- social icons -->
+            <div class="contact-section__social flex flex-1 text-xxs">
+              <div
+                class="contact-section__social-icon hover:bg-c200 hover:opacity-100 ml-0"
+              >
+                <a class="full-width-click" href="#" target="blank"
+                  ><i class="fab fa-twitter"></i
+                ></a>
+              </div>
+              <div
+                class="contact-section__social-icon hover:bg-c200 hover:opacity-100"
+              >
+                <a class="full-width-click" href="#" target="blank"
+                  ><i class="fab fa-facebook-square"></i
+                ></a>
+              </div>
+              <div
+                class="contact-section__social-icon hover:bg-c200 hover:opacity-100"
+              >
+                <a class="full-width-click" href="#" target="blank"
+                  ><i class="fab fa-pinterest-p"></i
+                ></a>
+              </div>
+              <div
+                class="contact-section__social-icon hover:bg-c200 hover:opacity-100"
+              >
+                <a class="full-width-click" href="#" target="blank"
+                  ><i class="fab fa-instagram"></i
+                ></a>
+              </div>
+            </div>
+
+            <!-- contact details -->
+            <div
+              class="contact-section__contacts flex flex-auto justify-end text-sm"
+            >
+              <div class="contact border-right">
+                <div class="contact-icon">
+                  <img
+                    class="h-7"
+                    src="assets/img/contact-footer/mail.png"
+                    alt="mail"
+                  />
+                </div>
+                <div class="information">
+                  <p class="leading-normal">needhelp@lovims.com</p>
+                  <small class="information-small leading-normal"
+                    >Email address</small
+                  >
+                </div>
+              </div>
+
+              <div class="contact border-right">
```

Comment:
give custom border color rgba(255,255,255,0.1) from psd

and padding left and right could be 3rem or closer

Reply:
done